resources/vbmc/Dockerfile

```diff
@@ -1,15 +1,13 @@
-ARG BASE_IMAGE=quay.io/centos/centos:stream9
+FROM docker.io/library/python:3.9.18-slim-bookworm
```

Comment:
Can we keep the BASE_IMAGE arg as it used to be, but defaulting to this python image?

Reply:
yes, I think I copy-pasted witout realizing I was removing it!